app/lib/world/world.coffee

```diff
@@ -316,6 +321,8 @@ module.exports = class World
     @addScripts level.scripts...
 
   loadClassFromCode: (js, name, kind='component') ->
+    window.box2d = require('lib/world/box2d') # TODO webpack: only load this when necessary
+    js = js.replace(/require\(/g, 'window.libWorldRequire(')
```

Comment:
👍 
This deals with DB code using "require" statements, now that there is no `window.require`

Reply:
Make sure this doesn't do something strange if user code/level default code has "require("